AzurePullRequests.Restore/Enums/RestoreType.cs

```diff
@@ -1,10 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
```

Comment:
Comment for test
![images](https://github.com/user-attachments/assets/77b34374-2b35-47d9-b60e-6288d3471770)